weblate_web/templates/base.html

```diff
@@ -89,7 +89,7 @@
                 <a href="https://status.weblate.org/" class="service-status">{% trans "Service Status" %}</a>
             </div>
 			<div class="footer-left">
-                Copyright © 2012—2019 <a href="https://cihar.com/">Michal Čihař</a>  •  {% trans "The software is Open Source under GNU GPLv3" %}
+                Copyright © 2012—2019 <a href="https://cihar.com/">Michal Čihař</a>  •  {% trans "The software is copylefted libre software, under GNU GPLv3+ license" %}
```

Comment:
This string needs to be kept shorter, it's in page footer.

How about just mentioning the license in full detail?

```suggestion
                Copyright © 2012—2019 <a href="https://cihar.com/">Michal Čihař</a>  •  {% trans "The software is under GNU GPLv3+ license" %}
```

Reply:
I went with "Software licensed GNU GPLv+3" Short and concise. It already scrolls past the other text
![Skjermbilde fra 2019-07-16 19 53 26](https://user-images.githubusercontent.com/13802408/61317596-946c3b00-a803-11e9-904a-7861517e3d2c.png)